### PR TITLE
Added EpiLogov2.0 extension

### PR DESCRIPTION
**Pull request by JakeSaunders1995**
Added Extension for epidemiological modelling

### CODE REVIEW

File: libraries.conf [hunk 1]
```diff
@@ -104,6 +104,16 @@ It allows you to perform unsupervised density-based clustering of turtles and pa
...
     downloadURL: "https://raw.githubusercontent.com/NetLogo/NetLogo-Libraries/6.1/extensions/encode-1.0.1.zip"
   }
+  {
+    name: "EpiLogo"
+    codeName: "EpiLogo"
+    shortDescription: "An epidemiological package for NetLogo "
+    longDescription: "An epidemiological package for NetLogo containing features regarding demographic population initialization, agent pathfinding, agent scheduling and disease campaign. This package allows for easier and less complex development of epidemiological models in NetLogo"
+    version: "2.0"
+    homepage: "https://github.com/JakeSaunders1995/EpiLogo"
+    downloadURL: "https://github.com/JakeSaunders1995/EpiLogo/tree/master/epilogo"
```
Comment (LaCuneta) [inline libraries.conf]: The `downloadURL` should point to the final location of the zip file in this repository, which would be: `https://raw.githubusercontent.com/NetLogo/NetLogo-Libraries/6.1/extensions/epilogo-2.0.0.zip`

File: libraries.conf [hunk 2]
```diff
@@ -104,6 +104,16 @@ It allows you to perform unsupervised density-based clustering of turtles and pa
...
+  }
+
```
Comment (LaCuneta) [inline libraries.conf]: Extra line break here can be removed.

**Review by LaCuneta — CHANGES_REQUESTED** [review]
Hey @JakeSaunders1995, thank you for your contribution.  I left a couple notes on changes to the `libraries.conf` file.  In addition, the zip file contains an inner `epilogo` folder that actually holds the `epilogo.jar` file.  That jar file should be in the zip file directly, not in a sub-folder.  A more visual description of this problem is [on our wiki page](https://github.com/NetLogo/NetLogo-Libraries/wiki/Submission-workflow#extension-submission-or-update).  

One more comment, not something you must change, but just an observation.  I see in your docs you separated primitive words with camel case, like `inputNewDemographic`.  NetLogo style is to use kebab case for word separation in primitives, like `input-new-demographic`.  Again, you don't have to change this, just pointing it out in case you were unaware.  

Once the zip file update and couple library file updates are made, we should be able to get this merged.  Thanks again.

**Comment by JakeSaunders1995** [comment]
I believe i have corrected all of the issues identified including the method name formatting to kebab case.

Kind regards
Jake

**Review by LaCuneta — APPROVED** [review]

**Pull request merged**